client/views/join_channel.tpl

```diff
@@ -0,0 +1,5 @@
+<form id="join-channel-{{id}}" class="join-form" method="post" action="" autocomplete="off">
+	<input type="text" class="input" name="channel" placeholder="Channel" pattern="[^\s]+" maxlength="200" title="Should be a valid channel name">
+	<input type="password" class="input" name="key" placeholder="Key (optional)" pattern="[^\s]+" title="Should be a valid channel key" maxlength="200" >
```

Comment:
Hmmm, should this be Password instead? I'm not sure "Key" makes it clear what it is.